sdk/ml/azure-ai-ml/azure/ai/ml/operations/_model_operations.py

```diff
@@ -248,7 +248,6 @@ def _get(self, name: str, version: Optional[str] = None) -> ModelVersion:  # nam
                     name=name,
                     version=version,
                     workspace_name=self._workspace_name,
-                    api_version="2023-02-01-preview",
```

Comment:
Hope this does not affect any model related feature ?

Reply:
There should not be a lock on the version from the model operations side. Otherwise, the Get Model wont show any of the new features added in the following versions. I think this was just a bug. Without removing setting this version, I dont get the new "stage" property in the response since that property was only added in the April preview version.